src/ProtoNN/ProtoNNPredictor.cpp

```diff
@@ -38,9 +140,126 @@ ProtoNNPredictor::ProtoNNPredictor(
 #endif
 }
 
+void ProtoNNPredictor::createOutputDirs()
```

Comment:
@t-chgupt @adityakusupati Can we agree on a common output dir name for ProtoNN and Bonsai and move this code to common